std/file.d

```diff
@@ -2318,7 +2319,7 @@ private bool ensureDirExists(in char[] pathname)
  * Throws: $(D FileException) on error.
  */
 
-void mkdirRecurse(in char[] pathname)
+void mkdirRecurse()(in char[] pathname)
```

Comment:
It will need to become a template anyway to accept arbitrary char ranges as all the other stuff in `std.file`.